src/creator/creator.controller.ts

```diff
@@ -153,4 +160,12 @@ export class CreatorController {
   remove(@Param('slug') slug: string) {
     return this.creatorService.remove(slug);
   }
+
+  @Post('follow/:slug')
+  async follow(
+    @WalletEntity() wallet: Wallet,
+    @Param('slug') slug: string,
+  ): Promise<boolean> {
+    return await this.walletCreatorService.follow(wallet.address, slug);
+  }
```

Comment:
I believe that you're missing "unfollow" endpoint alonside this one. Maybe use `toggleState` like we do with `WalletComic` and `WalletComicIssue`?